src/js/model.js

```diff
@@ -2,32 +2,32 @@
-    {name: 'Custom Airplane',  pos: 24, model: 'custom',     image: 'custom'},
-    {name: 'Custom Tricopter', pos: 25, model: 'custom',     image: 'custom'},
-    {name: 'Quad X 1234',      pos: 1,  model: 'quad_x',     image: 'quad_x_1234'}
+    { name: 'Tricopter',        pos: 3,  model: 'tricopter',  image: 'tri',          motors: 3, servos: true },
+    { name: 'Quad +',           pos: 2,  model: 'quad_x',     image: 'quad_p',       motors: 4, servos: false },
+    { name: 'Quad X',           pos: 0,  model: 'quad_x',     image: 'quad_x',       motors: 4, servos: false },
+    { name: 'Bicopter',         pos: 16, model: 'custom',     image: 'bicopter',     motors: 2, servos: true },
+    { name: 'Gimbal',           pos: 4,  model: 'custom',     image: 'custom',       motors: 0, servos: true },
+    { name: 'Y6',               pos: 20, model: 'y6',         image: 'y6',           motors: 6, servos: false },
+    { name: 'Hex +',            pos: 5,  model: 'hex_plus',   image: 'hex_p',        motors: 6, servos: false },
+    { name: 'Flying Wing',      pos: 10, model: 'custom',     image: 'flying_wing',  motors: 1, servos: true },
+    { name: 'Y4',               pos: 19, model: 'y4',         image: 'y4',           motors: 4, servos: false },
+    { name: 'Hex X',            pos: 6,  model: 'hex_x',      image: 'hex_x',        motors: 6, servos: false },
+    { name: 'Octo X8',          pos: 21, model: 'custom',     image: 'octo_x8',      motors: 8, servos: false },
+    { name: 'Octo Flat +',      pos: 8,  model: 'custom',     image: 'octo_flat_p',  motors: 8, servos: false },
+    { name: 'Octo Flat X',      pos: 9,  model: 'custom',     image: 'octo_flat_x',  motors: 8, servos: false },
+    { name: 'Airplane',         pos: 11, model: 'custom',     image: 'airplane',     motors: 1, servos: true },
+    { name: 'Heli 120',         pos: 12, model: 'custom',     image: 'custom',       motors: 1, servos: true },
+    { name: 'Heli 90',          pos: 13, model: 'custom',     image: 'custom',       motors: 0, servos: true },
+    { name: 'V-tail Quad',      pos: 17, model: 'quad_vtail', image: 'vtail_quad',   motors: 4, servos: false },
+    { name: 'Hex H',            pos: 7,  model: 'custom',     image: 'custom',       motors: 6, servos: false },
+    { name: 'PPM to SERVO',     pos: 22, model: 'custom',     image: 'custom',       motors: 0, servos: true },
+    { name: 'Dualcopter',       pos: 15, model: 'custom',     image: 'custom',       motors: 2, servos: true },
+    { name: 'Singlecopter',     pos: 14, model: 'custom',     image: 'custom',       motors: 1, servos: true },
+    { name: 'A-tail Quad',      pos: 18, model: 'quad_atail', image: 'atail_quad',   motors: 4, servos: false },
+    { name: 'Custom',           pos: 23, model: 'custom',     image: 'custom',       motors: 0, servos: false },
+    { name: 'Custom Airplane',  pos: 24, model: 'custom',     image: 'custom',       motors: 2, servos: true },
```

Comment:
What are these numbers based on?
For any 'custom' mixers the firmware does not actually define any motors by default, so these should probably default to 0 as well.

Reply:
These attributes are taken from firmware: `mixer_init.c` defines `const mixer_t mixers[]`